src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -1,30 +1,85 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Table from 'paragon/src/Table';
+import Button from 'paragon/src/Button';
 
-const AssetsTable = ({ assetsList }) => (
-  (!assetsList.length) ? (
-    <span>Loading....</span>
-  ) : (
-    <Table
-      columns={[
-        {
-          label: 'Name',
-          key: 'display_name',
-        },
-        {
-          label: 'Type',
-          key: 'content_type',
-        },
-        {
-          label: 'Date Added',
-          key: 'date_added',
-        },
-      ]}
-      data={assetsList}
-    />
-  )
-);
+import FontAwesomeStyles from 'font-awesome/css/font-awesome.min.css';
+
+class AssetsTable extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      finalAssetsList: [],
+    };
+
+    this.onDeleteClick = this.onDeleteClick.bind(this);
+    this.addDeleteButton = this.addDeleteButton.bind(this);
+  }
+
+  componentDidMount() {
```

Comment:
Do we need this blank method?

Reply:
Hmm actually -- do you think it would make sense to fire `addDeleteButton` in here too? Just in case the table gets initialized with all the props it needs and doesn't fire any `componentDidUpdate`s.

Reply:
I think that was the initial use of this method before I moved it to the `componentDidUpdate` - I can add another call into this method to catch both on initial load & if the list updates.